### ISSUE TITLE
Encodings requiring padding are not provided

### ISSUE THREAD

**Issue by andybalaam** [lilyinstarlight/i-dunno#7]
I'm not sure whether you consider this problem, or just something to document, but I realised that while [RFC 8771 section 3.1-2.2.2.3](https://www.rfc-editor.org/rfc/rfc8771.html#section-3.1-2.2.2.3) explains that encodings may include padding to allow more combinations of characters to match the size of IP addresses.

The current implementation only provides encodings that require no padding.

This limits the number of encodings provided, and in some cases reduces the available confusion level from the maximum possible.

Related: my Rust implementation has the same issue: https://gitlab.com/andybalaam/rust-i-dunno/-/issues/1

**Comment by lilyinstarlight** [comment]
I'll consider it a problem and see if I can look into doing that at some point. I remember reading that now when I initially read the RFC but I missed it when doing my implementation. The spec does say "MUST" and I don't think it'll be too bad since combinations where the end is padded can be added by modifying [lines 76 - 77](https://github.com/lilyinstarlight/i-dunno/blob/v0.1.3/i_dunno/__init__.py#L76-L77) in the `packed_combinations` function (and I don't think [lines 81 - 82](https://github.com/lilyinstarlight/i-dunno/blob/v0.1.3/i_dunno/__init__.py#L81-L82) need to change since if I remember right, that condition would only apply to areas before the end of the final address bitstring and I'm reading the RFC to say the padding bits go after all of the address bits).

**Comment by andybalaam** [comment]
Yeah, I also read it as padding goes after the meaningful bits.  I take it as somewhat optional, since the I-DUNNOs produced by the existing code are perfectly valid, just not selected from the full available range.

I do note that the number of possble encodings will increase vastly when this is implemented.
